src/components/v5/frame/PageLayout/PageLayout.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { FC, PropsWithChildren, useEffect, useRef } from 'react';
+import { useTablet } from '~hooks';
+
+import NavigationSidebar from '../NavigationSidebar';
+import { PageLayoutProps } from './types';
+
+const displayName = 'v5.frame.PageLayout';
+
+const PageLayout: FC<PropsWithChildren<PageLayoutProps>> = ({
```

Comment:
Hi @joanna-pagepro!

@arrenv asked me to chime in here. This PR looks good to me, generally! Lots of good work.

I just want to flag that as of now, this component (`PageLayout`) isn't used anywhere. In my PR I introduced a differentiation between a `ColonyLayout` (which is everything that is in the context of a Colony) and a `MainLayout` (which is everything else). Maybe you'd like to look into modifying these files instead.

Apart from that this is :+1:

Reply:
Thanks @chmanie! This is just the base for the new layout and will be used in the app in the next PR. I didn't want to make a big PR so that it would be easier for others to check it :) I'll check the files that you've created, but any changes will be applied in the following PR